### PR TITLE
Feature/extra observations

### PR DESCRIPTION
**Pull request by arturoLucazzz**
hi 👽 

- Im adding messages for the url fields 
- Im adding keys to avoid warnings on browser

### CODE REVIEW

File: web/src/components/DynamicFieldArray.tsx [hunk 1]
```diff
@@ -1,60 +1,42 @@
...
-                    className={styles.errorMsg}
-                  />
-                </div>
-              </div>
+              <LinkField
+                name={name}
+                index={index}
+                push={push}
+                remove={remove}
+                key={uuid()}
```
Comment (Wesitos) [inline web/src/components/DynamicFieldArray.tsx]: Wouldn't this key value change on every render? The `key` value is expected to be the same during the "lifetime" of the component. It tells React if a component has been "moved" (instead of being deleted) which allows it to correctly preserve it's state.

Given that we can only add and remove elements from the end of the listm providing the index as key (and add an [ignore-next-line](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1) for eslint) is a correct way of handling this.

tl;dr: We can use use the index as `key` because we cannot rearrange the list items.

File: web/src/views/Apply/FormQuestions.tsx [hunk 2]
```diff
@@ -9,26 +9,29 @@ export const FormQuestions = () => {
     firstName: Yup.string().required('Debes incluir tu nombre'),
     lastName: Yup.string().required('Debes incluir tu apellido'),
     email: Yup.string().email('Correo inválido').required('Debes incluir tu email'),
-    institution:  Yup.string().required('Debes incluir tu centro de estudios'),
+    institution: Yup.string().required('Debes incluir tu centro de estudios'),
```
Comment (Wesitos) [inline web/src/views/Apply/FormQuestions.tsx]: The institution field should not be a required field.

File: web/src/views/Apply/FormQuestions.tsx [hunk 3]
```diff
@@ -9,26 +9,29 @@ export const FormQuestions = () => {
...
     links: Yup.array()
       .of(
         Yup.object().shape({
-          url: Yup.string().required('Debes incluir tu link personal'),
+          url: Yup.string()
+            .url('Debes incluir un link válido')
+            .required('Debes incluir tu link personal'),
```
Comment (Wesitos) [inline web/src/views/Apply/FormQuestions.tsx]: @dguzt Should we keep the personal links as required? I think it should be optional. On the other hand, the form does not give us a lot of information about the candidate without this field. 

What do you think about adding some additional questions?

Comment (arturoLucazzz) [reply web/src/views/Apply/FormQuestions.tsx]: @dguzt  x2  xd

Comment (dguzt) [reply web/src/views/Apply/FormQuestions.tsx]: This question could be optional, but we can motivate the applicant to fill it to accelerate his/her apply process. We can use the following text as question description:
- "Puedes incluir uno o varios links personales que desees mostrarnos (GitHub, LinkedIn, página web personal, etc). Nos ayudarás mucho en tu proceso de postulación."

Text can change (talk to Oliver to check the ux writing).

Btw, the code link is required, but it's in the next question. Ok with that.

File: web/src/views/Apply/FormQuestions.tsx [hunk 4]
```diff
@@ -9,13 +9,12 @@ export const FormQuestions = () => {
...
     links: Yup.array()
       .of(
         Yup.object().shape({
           url: Yup.string()
-            .url('Debes incluir un link válido')
-            .required('Debes incluir tu link personal'),
+            .url('Debes incluir un link válido'),
         })
       )
       .min(1, 'Debes incluir al menos un link personal')
```
Comment (Wesitos) [inline web/src/views/Apply/FormQuestions.tsx]: We may also want to remove the "minimum" requirement.

File: web/src/views/Apply/FormQuestions.tsx [hunk 5]
```diff
@@ -62,14 +61,15 @@ export const FormQuestions = () => {
...
               <TitledField name="lastName" label="Apellidos *"/>
               <TitledField name="email" label="Correo electrónico *"/>
-              <TitledField name="institution" label="Centro de estudios *"/>
+              <TitledField name="institution" label="Centro de estudios"/>
             </div>
             <div className="grid grid-cols-1 gap-y-4">
               <h1 className="text-3xl font-bold mt-5">Preguntas</h1>
               <div>
-                <h2>Link personal *</h2>
+                <h2>Link personal</h2>
```
Comment (Wesitos) [inline web/src/views/Apply/FormQuestions.tsx]: I think we may want to pluralize this.
```suggestion
                <h2>Links personales</h2>
```
